client/routes/index.js

```diff
@@ -56,6 +57,14 @@ export default [
         path: '/reports',
         component: Reports
     },
+    {
+        path: '/test-plan-updater',
+        render: () => (
+            <ConfirmAuth requiredPermission="ADMIN">
+                <Route component={TestPlanUpdater} />
+            </ConfirmAuth>
+        )
+    },
```

Comment:
This might not be needed anymore, right?

Reply:
Right. I've removed it.